evap/results/templates/results_index_course.html

```diff
@@ -24,7 +24,7 @@
     <div data-col="responsible" data-order="{{ course.responsibles.first.last_name|lower }}">
         {{ course.responsibles_names }}
     </div>
-    <div data-col="result" data-order="{{ course.avg_grade }}">
+    <div data-col="result" data-order="{{ course.avg_grade|default:7 }}">
```

Comment:
As I fetch the key via `.data("order")`, jQuery converts the values into an integer. But, a `"None"` keeps a `"None"`. To have consistent sorting behavior, I convert `None` to an imaginary grade. Possibly this should not be done in the template but in the tools.

Yes, in Js both `"None" >= 4` and `"None" <= 4` are `false`…

Reply:
to have the default there is fine for me